src/components/InputItem/InputItem.module.css

```diff
@@ -1,4 +1,9 @@
 .wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/react-todo/pullRequest?prid=5576112)